src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/Result/HealthCertificateValidationPassedViewModel.swift

```diff
@@ -52,11 +52,15 @@ struct HealthCertificateValidationPassedViewModel: HealthCertificateValidationRe
 					.bulletPoint(text: AppStrings.HealthCertificate.Validation.Result.Passed.hint2, spacing: .large),
 					.bulletPoint(text: AppStrings.HealthCertificate.Validation.Result.Passed.hint3, spacing: .large),
 					.bulletPoint(text: AppStrings.HealthCertificate.Validation.Result.Passed.hint4, spacing: .large),
-					.dynamicType(
-						text: """
-							<p>\(AppStrings.HealthCertificate.Validation.Result.moreInformation01) <a href="\(AppStrings.Links.healthCertificateValidationFAQ)">\(AppStrings.HealthCertificate.Validation.Result.moreInformation02)</a> \(AppStrings.HealthCertificate.Validation.Result.moreInformation03) <a href="\(AppStrings.Links.healthCertificateValidationEU)">\(AppStrings.Links.healthCertificateValidationEU)</a>.</p>
-							""",
-						cellStyle: .htmlString
+					.textWithLinks(
```

Comment:
What about using the already implemented style attribute of a body cell?
`.textView(.link)` sets `UIDataDetectorTypes`.

```
					.body(
						text: AppStrings.HealthCertificate.Validation.body3,
						style: .textView(.link),
						accessibilityIdentifier: ""
					),
```

Reply:
Ok, I See... there is a placeholder for the link